kvmagent/kvmagent/plugins/vm_local_volume_cache/command_wrapper/filesystem.py

```
@@ -0,0 +1,263 @@
+        mount_points = FileSystemCommandWrapper.parse_findmnt_output(cmd.stdout)
+        if not mount_points:
+            return None
+        return mount_points.pop()
+        
+    @staticmethod
+    def mount_filesystem(filesystem_uuid, mount_path, force=False):
+        # type: (str, str, bool) -> None
+        if FileSystemCommandWrapper.get_mount_point(filesystem_uuid, mount_path):
+            return
+        if not os.path.exists(mount_path):
+            os.makedirs(mount_path, 0x0775)
```

Ruff (0.15.2)

[warning] Non-octal mode

(RUF064)